catalog/core/catalog-core-validationparser/src/test/groovy/ddf/catalog/validation/impl/ValidationParserSpecTest.groovy

```diff
@@ -118,35 +109,21 @@ class ValidationParserSpecTest extends Specification {
         setup:
         file.withPrintWriter { it.write(defaultValues) }
         mockStatic(FrameworkUtil.class)
-        when(FrameworkUtil.getBundle(ValidationParser.class)).thenReturn(Mock(Bundle))
-
+        Bundle mockBundle = Mock(Bundle)
+        when(FrameworkUtil.getBundle(ValidationParser.class)).thenReturn(mockBundle)
+        BundleContext mockBundleContext = Mock(BundleContext)
+        mockBundle.getBundleContext() >> mockBundleContext
         when:
         validationParser.install(file)
 
         then:
-        verifyDefaultValue("type2", "title", "Default Title")
-        def expectedDateTime = OffsetDateTime.of(2020, 2, 2, 2, 2, 2, 0, ZoneOffset.UTC)
-        verifyDefaultValue("expiration", Date.from(expectedDateTime.toInstant()))
-        verifyDefaultValue("thumbnail", [0x41, 0x42, 0x43] as byte[])
-        verifyDefaultValue("short", -123)
-        verifyDefaultValue("type1", "integer", 1234567890)
-        verifyDefaultValue("long", 1125899906842624)
-        verifyDefaultValue("type1", "float", -90.912f)
-        verifyDefaultValue("type2", "float", -90.912f)
-        verifyDefaultValue("double", 84812938.293818)
-        verifyDefaultValue("boolean", true)
-    }
-
-    void verifyDefaultValue(attributeName, expected) {
-        Optional<Serializable> optional = defaultAttributeValueRegistry.getDefaultValue("", attributeName)
-        assert optional.isPresent()
-        assert optional.get() == expected
-    }
-
-    void verifyDefaultValue(metacardTypeName, attributeName, expected) {
-        Optional<Serializable> optional = defaultAttributeValueRegistry.getDefaultValue(metacardTypeName, attributeName)
-        assert optional.isPresent()
-        assert optional.get() == expected
+        1 * mockBundleContext.registerService(DefaultAttributeValue.class, _ as DefaultAttributeValue, {
```

Comment:
We should probably use predicates on the second argument to verify that the right `DefaultAttributeValue`s are being registered.

For example:
```
1 * mockBundleContext.registerService(DefaultAttributeValue.class, {
        it.getMetacardTypeNames().isEmpty()
        it.getAttributeName() == "short"
        it.getDefaultValue() == -123
    }, {
        it.get("attribute") == "short"
})